staging/src/k8s.io/kube-scheduler/config/v1beta2/types_pluginargs.go

```diff
@@ -178,6 +178,12 @@ type VolumeBindingArgs struct {
 	// Value must be non-negative integer. The value zero indicates no waiting.
 	// If this value is nil, the default value (600) will be used.
 	BindTimeoutSeconds *int64 `json:"bindTimeoutSeconds,omitempty"`
+
+	// Shape specifies the shape points defining score function shape.
+	// +featureGate=VolumeCapacityPriority
+	// +optional
+	// +listType=atomic
+	Shape []UtilizationShapePoint `json:"shape,omitempty"`
```

Comment:
why not add it to v1beta1 too? also, now that we removed the defaulting that happens in the plugin, v1beta1 users will not get the old default behavior.

Reply:
ack, I'll add it to v1beta1.

Reply:
> this wasn't addressed

I may not understand this correctly. I've added this field and defaulting behavior in v1beta1 (`staging/src/k8s.io/kube-scheduler/config/v1beta1/{types_pluginargs.go,defaults.go}`). Do I missing something else?

Reply:
this is referring to the comment you resolved: https://github.com/kubernetes/kubernetes/pull/99594#discussion_r654459750